pkg/node/Dockerfile.build

```diff
@@ -2,7 +2,4 @@ FROM    ubuntu:18.04
 
 ADD     health_probe    health_probe
 
-# Remove bash packet to get rid of related CVEs
```

Comment:
Same here

Reply:
21.04 Not vulnerable accouding to https://ubuntu.com/security/CVE-2019-18276 
![image](https://user-images.githubusercontent.com/88684715/144378115-9a14dc29-2e8e-4e72-94f9-c1adb8189b31.png)